query/workmanager.go

```diff
@@ -377,6 +379,11 @@ Loop:
 					result.job.timeout = newTimeout
 				}
 
+				// Refresh peer rank on disconnect.
+				if result.err == ErrPeerDisconnected {
+					w.cfg.Ranking.ResetRanking(result.peer.Addr())
```

Comment:
Line should fold to 80 char columns.

Reply:
Please I checked this in my IDE and it is less than 80 char column

Reply:
@Chinwendu20 did you configure your IDE to use 8 spaces for a tab character? If not, this will probably pop up in future PRs as well (in `lnd` the line length is even enforced by a linter).

Reply:
Thanks for pointing this out, this was one of my earlier PRs. I have now configured my IDE to this. Does it still appear as going beyond the limit?

Reply:
This particular line as it was merged, yes. But no big deal, just wanted to let you know for future PRs.